oauth2client/service_account.py

```diff
@@ -461,3 +465,96 @@ def create_delegated(self, sub):
         result._private_key_pkcs12 = self._private_key_pkcs12
         result._private_key_password = self._private_key_password
         return result
+
+
+def _datetime_to_secs(utc_time):
+   # time_delta.total_seconds() not supported in Python 2.6
```

Comment:
I know you are, and the exception that I'm trying to carve out is for references to issues.

I agree that bare `TODO`s are bad. I agree that `TODO(username)` is bad. I hope I can persuade you to love `TODO(issue reference)` as just a little frosting on the cake of the world-class issue tracker.

Imagine someone coming to this code and just seeing something that looks sub-optimal. They might go to the issue tracker and file an issue about it. They might draft a pull request. They might go to the issue tracker and search and find our issue about it. But by cross-referencing our issue here in the code with a `TODO(issue <number>)`, we get to tell them that we already know about the problem and have a work item filed (and _filed in the right place_) for it.

Perhaps another part of the trick to liking these is that they're just pointers and they don't take away from the issue tracker being the primary store of work to be done. A bare `TODO` or a `TODO(<username>)` is a problem because it's the only record of work to be done, and it's not indexed or landed on anyone's product roadmap or anything; it's more or less lost until it's rediscovered.

Is that any help?